.clang-format

```diff
@@ -160,7 +160,7 @@ SpacesInParentheses: false
 SpacesInSquareBrackets: false
 SpaceBeforeSquareBrackets: false
 BitFieldColonSpacing: Both
-Standard:        c++14
+Standard: c++20
```

Comment:
Latest is c++20, which supports c++23.
https://clang.llvm.org/docs/ClangFormatStyleOptions.html#standard